adam-core/src/main/scala/org/bdgenomics/adam/util/FileExtensions.scala

```diff
@@ -63,7 +63,11 @@ private[adam] object FileExtensions {
    */
   def isFastqExt(pathName: String): Boolean = {
     pathName.endsWith(".fq") ||
-      pathName.endsWith(".fastq")
+      pathName.endsWith(".fastq") ||
+      pathName.endsWith(".fastq.gz") ||
```

Comment:
On a related note, the doc [here](https://github.com/fnothaft/adam/blob/cefab0105d1948c9724ed0eb88617dc6e8de0657/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala#L2646) and similar in related methods should be updated.  Is bz2 still supported?  Other user-configured Hadoop compression codecs?